### PR TITLE
some trader stuff and damage reducing

### PR DESCRIPTION
**Pull request by Grohiik**
This will merge conflict, but i think i can fix it

### CODE REVIEW

File: mappings/net/minecraft/enchantment/EnchantmentHelper.mapping [hunk 1]
```diff
@@ -17,6 +21,8 @@ CLASS net/minecraft/class_66 net/minecraft/enchantment/EnchantmentHelper
 		ARG 1 stack
 	METHOD method_108 getEnchantments (Lnet/minecraft/class_2056;)Ljava/util/Map;
 		ARG 0 stack
+	METHOD method_110 EnchantmentResistance ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
```
Comment (SpaceWalkerRS) [inline mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:24]: should this be `getEnchantmentResistance` maybe?

Comment (Grohiik) [reply mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:32]: most likely

File: mappings/net/minecraft/enchantment/EnchantmentHelper.mapping [hunk 2]
```diff
@@ -31,9 +33,9 @@ CLASS net/minecraft/class_66 net/minecraft/enchantment/EnchantmentHelper
 		COMMENT Returns the increase in damage to the target due to the attacker's enchantments.
 		ARG 0 attacker
 		ARG 1 target
-	METHOD method_110 getEnchantmentResistance ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
-		COMMENT Returns the increase in protection from the damage source due to the armor stacks' enchantments.
-		ARG 0 armor
+	METHOD method_110 modifyProtection ([Lnet/minecraft/class_2056;Lnet/minecraft/class_1733;)I
+		COMMENT Returns the increase in protection from the damage source due to the item stacks' enchantments.
+		ARG 0 stacks
```
Comment (Grohiik) [inline mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:38]: This is a change back too space's name.

it is only ever called for armor. I think my name is better

File: mappings/net/minecraft/enchantment/EnchantmentHelper.mapping [hunk 3]
```diff
@@ -1,13 +1,15 @@
 CLASS net/minecraft/class_66 net/minecraft/enchantment/EnchantmentHelper
+	COMMENT Applies a modifier too the stack.
```
Comment (SpaceWalkerRS) [inline mappings/net/minecraft/enchantment/EnchantmentHelper.mapping:2]: Duplicate comment? Also slight typo "too" -> "to".

File: mappings/net/minecraft/entity/living/LivingEntity.mapping [hunk 4]
```diff
@@ -109,12 +109,14 @@ CLASS net/minecraft/class_1752 net/minecraft/entity/living/LivingEntity
 	METHOD method_7096 dropEquipment (ZI)V
 		ARG 1 hasBeenHit
 		ARG 2 lootingLevel
-	METHOD method_7097 initAttributes ()V
-	METHOD method_7098 canBreatheUnderWater ()Z
+	METHOD method_7097 initializeAttributes ()V
```
Comment (SpaceWalkerRS) [inline mappings/net/minecraft/entity/living/LivingEntity.mapping:112]: "initialize" -> "init" as per the style guide.

**Pull request merged**